_maps/shuttles/shiptest/nanotrasen_skipper.dmm

```diff
@@ -1,5291 +1,7101 @@
+	id = "noiwillnotgiveyouaa"
+	},
+/turf/open/floor/plating,
+/area/ship/crew/crewthree)
+"co" = (
+/obj/item/radio/intercom{
+	pixel_y = 25
+	},
+/turf/open/floor/carpet/nanoweave,
+/area/ship/hallway/central)
+"cp" = (
+/obj/structure/sign/directions/medical{
+	dir = 4
+	},
+/obj/structure/sign/directions/command{
 	dir = 4;
-	name = "Helm"
+	pixel_y = -6
 	},
-/obj/effect/turf_decal/corner/opaque/blue{
-	dir = 1
+/turf/closed/wall,
+/area/ship/crew/cryo)
+"cE" = (
+/obj/machinery/door/airlock/public/glass/incinerator/atmos_interior,
+/obj/structure/disposalpipe/segment{
+	dir = 8
 	},
-/obj/effect/turf_decal/corner/opaque/blue{
+/obj/machinery/door/firedoor/border_only{
 	dir = 8
 	},
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
-	dir = 1
+/obj/machinery/door/firedoor/border_only{
+	dir = 4
 	},
-/turf/open/floor/plasteel/dark,
-/area/ship/bridge)
-"ce" = (
-/obj/machinery/light{
-	dir = 1
+/turf/open/floor/engine,
+/area/ship/engineering/engine)
+"cF" = (
+/obj/machinery/atmospherics/pipe/simple/cyan/hidden{
+	dir = 4
 	},
-/obj/effect/turf_decal/corner/opaque/white/mono,
-/turf/open/floor/plasteel/mono/white,
-/area/ship/crew/canteen)
-"ci" = (
-/obj/machinery/atmospherics/pipe/simple/cyan/visible,
-/obj/machinery/atmospherics/pipe/simple/green/visible{
+/turf/closed/wall/r_wall,
+/area/ship/crew/cryo)
+"cL" = (
+/obj/machinery/atmospherics/components/binary/valve/digital{
 	dir = 4
 	},
-/obj/effect/turf_decal/industrial/warning,
-/turf/open/floor/plasteel,
-/area/ship/engineering/atmospherics)
-"co" = (
-/obj/structure/table/reinforced,
-/obj/item/reagent_containers/food/condiment/saltshaker{
-	pixel_x = -8;
-	pixel_y = 5
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
 	},
-/obj/item/reagent_containers/food/condiment/peppermill{
-	pixel_x = -8
+/obj/machinery/atmospherics/pipe/simple/scrubbers/visible,
+/obj/effect/turf_decal/techfloor{
+	dir = 8
 	},
-/obj/machinery/door/firedoor,
-/turf/open/floor/plasteel/dark,
-/area/ship/crew/canteen)
-"cr" = (
-/turf/open/floor/plating,
+/turf/open/floor/plasteel/tech/techmaint,
 /area/ship/engineering)
-"cx" = (
-/obj/machinery/vending/clothing,
-/turf/open/floor/plasteel/dark,
-/area/ship/hallway/central)
-"cy" = (
-/obj/structure/cable{
-	icon_state = "4-8"
-	},
-/obj/machinery/light_switch{
-	pixel_x = -25;
-	pixel_y = 25
+"cQ" = (
+/obj/effect/turf_decal/corner/opaque/black{
+	dir = 9
 	},
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
 	dir = 4
 	},
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
 	dir = 4
 	},
-/turf/open/floor/wood,
-/area/ship/crew/solgov)
-"cz" = (
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/turf/open/floor/plasteel,
+/area/ship/cargo/office)
+"cR" = (
+/obj/structure/closet/secure_closet/wall{
+	icon_state = "sec_wall";
+	name = "ammunition locker";
+	pixel_y = -28;
+	req_access_txt = "57, 41";
```

Comment:
These lockers are currently configured to simultaneously require HoP and Quartermaster access to open on a ship that has neither position on the roster. Functionally speaking this means only the Captain can open them. This isn't necessarily a problem, just a little odd.